python/src/otel/otel_sdk/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -30,7 +30,7 @@
     )
 
     # Enable instrumentation
-    AwsLambdaInstrumentor().instrument()
+    AwsLambdaInstrumentor().instrument(skip_dep_check=True)
```

Comment:
Can we use flag `skip_dep_check` to remove [these code](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/python/src/otel/otel_sdk/otel_wrapper.py#L21-L47) ?

Reply:
Hm I'm not sure what we would replace it with. The `_load_distros`, `_load_instrumentors` and `_load_configurators` all initialize the classes they find. Normally, its important to **not** `skip_dep_check` because the classes `_load_instrumentors` loads **requires** that the dependencies are installed. Attempting to instrument without the dependencies would cause an import error.

I think it might be possible to use the upstream `opentelemetry-instrumentation` package as it is today instead of doing this code ourselves, but we don't need that in this PR.

Reply:
Let's merge the code first. I feel if is_recording() is essential why it is not integrated inside of start_span. I can raise issue in python repo later.